src/components/Page/DefaultSEO/index.tsx

```diff
@@ -76,21 +95,23 @@ const DefaultSEO: React.FC<IDefaultSEOProps> = ({ pageInfo }) => {
       meta={meta}
       link={[
         {
-          rel: 'shortcut icon',
+          rel: 'icon',
           type: 'image/vnd.microsoft.icon',
-          href: '/favicon.ico'
+          href: siteUrl + '/favicon.ico'
         },
         {
           rel: 'icon',
           type: 'image/png',
-          href: '/favicon-32x32.png',
-          sizes: '32x32'
+          href: siteUrl + '/favicon-32x32.png'
         },
         {
           rel: 'icon',
           type: 'image/png',
-          href: '/favicon-16x16.png',
-          sizes: '16x16'
+          href: siteUrl + '/favicon-32x32.png'
```

Comment:
16x16? do we need more sizes? we need to do this if we understand why do we have them.

Reply:
Oops! Looks like I kinda messed up this manual re-add without a quick `git diff` to lean on.
I just found this:
https://support.google.com/webmasters/answer/9290858?hl=en
Seems like the original change that removed them was made based on this info, which provides guidelines to have one favicon at 48x48 (or a larger multiple of that) and explicitly says not to provide a 16x16 icon.
It also says to use the `rel='shortcut icon` bit removed from before, so I'll re-add that.
I'll adapt the PR to this new info.

Reply:
yep, that link makes sense, but I would not do SEO in isolation from other needs. meta tags exist not only for Google search page, also link sharing, browsers, embeds, etc, etc - that's why reviewing meta tags ticket ticket is relevant to this one.

Reply:
You're right, and the article itself says that "penalized" sites weren't downranked, simply not allowed to use their custom 16x16 icon.

This should mean that despite Google ignoring it and downscaling a larger image, having the 16x16 (and any other sizes) shouldn't cause any harm. I'll keep the old entries exactly how they are and re-add `shortcut` to the `.ico` entry as it seems like the most proper one.

Making these favicons absolute is probably premature as well, since favicons work now. I now think these favicons should just be reverted to their original state, which I'll do next commit.

Reply:
@rogermparent my only point is that this ticket + meta tags one requires a bit of research :) what are the best practices now? what images and for what should we include, etc, etc.

Reply:
I get what you mean. I generally use Google Lighthouse as the source of truth when if comes to SEO because SEO is more or less for Google. Since Lighthouse doesn't flag having all our original icon sizes as a problem, I think it's best for this PR to keep them to remove later.
Since they were removed in the first commit from @fabiosantoscode, I'll tag him to ask the reason.
I think not removing any is the best for this PR so it can go through easier and fix the share card image issue, which this PR appears to do now from my tests of a new build on it with both the Facebook debugger and another at http://debug.iframely.com for good measure.

[The opengraph spec](https://ogp.me/#metadata) shows that they expect urls to be absolute instead of root-relative, which is where I believe the issue with share card images comes from.